pycryptobot.py

```diff
@@ -228,7 +228,8 @@ def executeJob(sc, app=PyCryptoBot(), state=AppState(), trading_data=pd.DataFram
     else:
         current_df_index = state.last_df_index
 
-    formatted_current_df_index = f'{current_df_index} 00:00:00' if len(current_df_index) == 10 else current_df_index
+    candle_time = f'{current_df_index} 00:00:00' if len(current_df_index) == 10 else current_df_index
+    formatted_current_df_index = str(datetime.utcfromtimestamp(float(datetime.strptime(candle_time, '%Y-%m-%d %H:%M:%S').strftime("%s"))))
```

Comment:
I'm afraid I don't understand the trick that's happening here - it seems to me the TZ is UTC here since that's what `datetime.utcfromtimestamp` will return. How does it fix the timestamp to not be printed as UTC?

Reply:
@letsautom8 would you mind sharing a wider snapshot of the output? I can't shrug off a feeling your fix actually converts a DF time to a UTC timestamp...

Reply:
you're right. i got it reversed. working on it.